.code-samples.meilisearch.yaml

```diff
@@ -578,8 +593,26 @@ get_version_1: |-
           print(error)
       }
   }
+get_pretty_sys_info_1: |-
```

Comment:
Im not sure they still exist: https://github.com/meilisearch/documentation/blob/master/.code-samples.meilisearch.yaml